celery/utils/imports.py

```diff
@@ -51,8 +51,13 @@ def instantiate(name, *args, **kwargs):
 @contextmanager
 def cwd_in_path():
     """Context adding the current working directory to sys.path."""
-    cwd = os.getcwd()
-    if cwd in sys.path:
+    try:
```

Comment:
could you please add some unit test to verify the change?

Reply:
Unit test added